### PR TITLE
fix echodoc don't work in elixir.

### PR DESCRIPTION
**Pull request by sjava**

### CODE REVIEW

**Comment by Shougo** [comment]
If not, I will close it.

**Comment by Shougo** [comment]
You must provide the example and `word` and `comp` value in Elixir.

Like the comment.

```
    " Completion 'word' is what actually completed, if the parsed name is
    " different, it's probably because 'info' is an abstract function
    " signature.  .e.g in Go:
    " completed: BoolVar(p *bool, name string, value bool, usage string)
    " info:      func(p *bool, name string, value bool, usage string)
```

**Comment by sjava** [comment]
when input  List.insert
comp: {'opens': [0, 0, 0], 'pos': -1, 'name': 'insert_at', 'end': 27, 'ppos': 0, 'i': 0, 'trailing': '', 'start': 10, 'args': ['list', ' index', ' value']}
word: List

**Comment by Shougo** [comment]
Please upload `v:completed_item` value after completion.

File: autoload/echodoc/util.vim [hunk 1]
```diff
@@ -278,7 +278,7 @@ function! echodoc#util#completion_signature(completion, maxlen, filetype) abort
 
   let comp = stack[0]
   let word = matchstr(a:completion.word, '\k\+')
-  if comp.name !=# word
+  if comp.name !=# word && a:filetype != 'elixir'
```
Comment (Shougo) [inline autoload/echodoc/util.vim]: It must be `a:filetype !=# 'elixir'`

**Comment by sjava** [comment]
when input List.insert() ,v:completed_item is:
{'word': 'List.insert_at', 'menu': '[alchemist]  List', 'info': '', 'kind': 'f', 'abbr': 'insert_at(list, index, value)'}

File: autoload/echodoc/util.vim (hunk 1 above)
Comment (sjava) [reply autoload/echodoc/util.vim]: sorry, i'am a newbie in vimscript.

**Comment by Shougo** [comment]
OK.  I get the problem.

**Pull request closed without merging**